src/style.css

```diff
@@ -1 +1,61 @@
-/* Styles go here */
+* {
```

Comment:
use * only for box-sizing, move font props to html tag and spacing you can reset for required tags directly

Reply:
check